Loop/View Controllers/StatusTableViewController.swift

```diff
@@ -251,11 +251,14 @@ final class StatusTableViewController: LoopChartsTableViewController {
     }
 
     private func registerPumpManager() {
-        if let pumpManager = deviceManager.pumpManager {
-            self.basalDeliveryState = pumpManager.status.basalDeliveryState
-            pumpManager.removeStatusObserver(self)
-            pumpManager.addStatusObserver(self, queue: .main)
+        guard let pumpManager = deviceManager.pumpManager else {
```

Comment:
[nit] The guard could be removed, and the pump manager accessed with optional chaining below.